src/features/query-bar/components/filter/Condition.js

```diff
@@ -109,22 +110,46 @@ const Condition = ({
         return [];
     };
 
+    const checkValidValue = (values: Array<ConditionValueType>, type: string) => {
+        let isValueInvalid = false;
```

Comment:
It seems weird that we're returning `isValueInvalid`.
With function named `checkValidValue`, I think the natural assumption will be that it returns true if the value is valid.